src/service.js

```diff
@@ -333,7 +335,7 @@ function getServicePromiseInternal(holder, id) {
 /**
  * @param {!Object} holder
  * @param {string} id of the service.
- * @return {?Promise<*>}
+ * @return {?Promise}
```

Comment:
done. I do want to make sure that `!Object` is correct here because in `getElementServiceIsAvailable` the return type eventually becomes `?Object` (probably because we return null there explicitly inside a promise after some conditions)

cc @cramforce